streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/TopologyTest.scala

```diff
@@ -276,61 +274,6 @@ class TopologyTest {
     assertEquals(getTopologyScala, getTopologyJava)
   }
 
-  @nowarn
-  @Test
-  def shouldBuildIdenticalTopologyInJavaNScalaTransform(): Unit = {
```

Comment:
Wondering if we have a test gap, as we don't have a similar test for `process()` -- should we add a new test for `#process()` like this one (or rewrite this test for `#process()`)?

Reply:
I have rewrote this test with `process`, but have missed to rewrite the test name. I will fix it